sentences/en/light_HassTurnOff.yaml

```diff
@@ -28,3 +28,13 @@ intents:
         slots:
           domain: "light"
           name: "all"
+
+      # Turn off all lights in the same area as a satellite device
+      - sentences:
+          - "<turn> off all[ the| my] lights"
```

Comment:
I feel as though "my lights" refers to all of them, not just in the area.

I'd also add an optional `[ here]` at the end

Reply:
I was imagining "turn off my lights" being used in the sense of "turn off the lights in **my** office"

Reply:
I won't argue with that, it makes sense in a lot of contexts. If the case may be, we will receive issues and we can improve after the fact.

But i feel like `[ [in ]here]` at the end would be a good addition.